lib/ain-grpc/src/rpc/eth.rs

```diff
@@ -293,12 +294,17 @@ impl MetachainRPCModule {
                         .checked_sub(U256::from(finality_count))?;
                     self.handler.storage.get_block_by_number(&safe_block_number)
                 })
-            }
+            },
```

Comment:
Remove changes here which fails rust lint check

Reply:
Resolved in [e7668d5](https://github.com/DeFiCh/ain/pull/2324/commits/e7668d577a89d0f85127fa30efc090f635530d3c)